docs/jwst/error_propagation/main.rst

```diff
@@ -0,0 +1,374 @@
+single combined 2D or 3D spectral product and a combined 1D spectrum.  The steps 
+in this pipeline are master_background, exp_to_source, mrs_match, 
+outlier_detection, resample_spec, cube_build, extract_1d, and combine_1d.  The
+paricular steps executed depend on particular instruments or instrument modes; 
+for details see `Calwebb_spec3
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_spec3.html>`_
+
+Of the steps listed above, below are the steps that either use or modify
+variance/error arrays that are propagated through these pipelines.
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Cube_build
+~~~~~~~~~~
+Cube_build takes MIRI or NIRSpec IFU calibrated 2-D images and produces 3-D
+spectral cubes.  In the output spectral cube, the SCI exension contains the
+surface brightness of cube spaxels in units of mJy/arcsecond^2, and the ERR
+extension contains the uncertainty on the SCI values.
+
+
+Stage 3 Coronographic Processing Pipeline - calwebb_coron3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The stage 3 coronagraphic pipeline is to be applied to associations of
+calibrated NIRCam coronagraphic and MIRI Lyot and 4QPM exposures, and is used 
+to produce PSF-subtracted, resampled, combined images of the source object.  The
+steps in this pipeline are stack_refs, align_refs, klip, outlier_detection, and
+resample.  The only one of these steps that either uses or modifies variance/error
+arrays that are propagated through the pipeline is outlier_detection, described below.
+For more details on this pipeline see `Calwebb_coron3
+<https://jwst-pipeline.readthedocs.io/en/latest/jwst/pipeline/calwebb_coron3.html>`_
+
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays. 
+
+
+Stage 3 Time-Series Observation (TSO) Processing Pipeline - calwebb_tso3
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+The stage 3 TSO pipeline is to be applied to associations of calibrated TSO
+exposures.  The steps in this pipeline are outlier_detection, tso_photometry,
+extract_1d, and white_light.  The only one of these steps that either uses or
+modifies variance/error arrays that are propagated through the pipeline is
+outlier_detection, described here:
+
+Outlier_detection 
+~~~~~~~~~~~~~~~~~
+This step uses the ERR array to make a local noise model, based on the readnoise
+and calibration errors of earlier steps in the pipeline. This step does not modify
+the ERR array or any of the VAR arrays.
+
+Stage 3 WFS&C Processing pipeline - calwebb_wfs-image3
```

Comment:
Change this heading to just "wfs_combine", like with "outlier_detection" and "cube_build" above and then modify the following text in a suitable way to mention that the "wfs_combine" step is only applied in the Stage 3 Wavefront Sensing and Control (calwebb_wfs-image3) pipeline.